### DISCUSSION TITLE
Tested today BrosTrend AC3L confirmed working, Netgear A6210 NOT working (Monterey 12.1)

### DISCUSSION THREAD

**Discussion by savisch** [chris1111/Wireless-USB-OC-Big-Sur-Adapter#192] [General]
Tested on macbook pro 15 running monterey 12.1.
Brostrend AC3L is working. 
Also tested a Netgear A6210 and that is NOT working.

**Comment by chris1111** [comment]
Thanks to report
[Netgear A6210](https://www.netgear.com/support/product/A6210.aspx#download) support only in Windows
